Formula/clang-format.rb

```diff
@@ -1,20 +1,11 @@
 class ClangFormat < Formula
   desc "Formatting tools for C, C++, Obj-C, Java, JavaScript, TypeScript"
   homepage "https://clang.llvm.org/docs/ClangFormat.html"
-  version "2019-05-14"
+  version "2019-12-20"
 
   stable do
-    depends_on "subversion" => :build
-    url "https://llvm.org/svn/llvm-project/llvm/tags/google/stable/2019-05-14/", :using => :svn
-
-    resource "clang" do
-      url "https://llvm.org/svn/llvm-project/cfe/tags/google/stable/2019-05-14/", :using => :svn
-    end
-
-    resource "libcxx" do
-      url "https://releases.llvm.org/9.0.0/libcxx-9.0.0.src.tar.xz"
-      sha256 "3c4162972b5d3204ba47ac384aa456855a17b5e97422723d4758251acf1ed28c"
-    end
+    depends_on "git" => :build
+    url "https://github.com/llvm/llvm-project.git", :tag => "llvmorg-9.0.1"
```

Comment:
This needs to have a revision specified too, to prevent retagging

Reply:
you mean stick with SHA?

Reply:
See this: https://github.com/Homebrew/homebrew-core/blob/master/Formula/cling.rb#L4-L6